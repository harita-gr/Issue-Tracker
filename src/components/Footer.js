import React from 'react'
import { Container, Row, Col } from 'react-bootstrap'

function Footer() {
    return (
        // <div className='footer'>
        //     <h3>Copyright &copy; 2021 @ Harita Ravindranath </h3>
        // </div>
        <footer>
        <Container>
          <Row>
            <Col className='text-center'> <h6>Copyright &copy; 2021 @ Harita Ravindranath </h6></Col>
          </Row>
        </Container>
      </footer>
    )
}

export default Footer
