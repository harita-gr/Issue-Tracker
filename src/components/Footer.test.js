import React from 'react';
import {shallow } from 'enzyme';
import Footer from './Footer'

describe('<Footer/>', () => {

    let wrapper;

     
        it('should render the footer text correctly', () => {
            wrapper = shallow(<Footer/>); 
            expect(wrapper.find("h6").text()).toContain("Copyright")
        });

        
      it('render the footer correctly',() =>{      
        wrapper = shallow(<Footer/>); 
        expect(wrapper).toMatchSnapshot();
      })

});