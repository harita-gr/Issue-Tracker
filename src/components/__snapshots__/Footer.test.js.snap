// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> render the footer correctly 1`] = `
<footer>
  <Container
    fluid={false}
  >
    <Row
      noGutters={false}
    >
      <Col
        className="text-center"
      >
         
        <h6>
          Copyright Â© 2021 @ Harita Ravindranath 
        </h6>
      </Col>
    </Row>
  </Container>
</footer>
`;
